### PR TITLE
Add Login function and logout

### PR DESCRIPTION
**Pull request by QuocThangPhu**
Create Login and Logout

### CODE REVIEW

**Comment by khajackie2206** [comment]
your code is very cool <3

File: templates/lucky/number.html.twig [hunk 1]
```diff
@@ -1 +1 @@
-<h1>Your lucky number is {{ number }}</h1>
\ No newline at end of file
+<h1>Your lucky number is {{ number }}</h1>
```
Comment (richardktran) [inline templates/lucky/number.html.twig:1]: Why lucky number feature in the login feature?

Comment (QuocThangPhu) [reply templates/lucky/number.html.twig:1]: because i had some error in main and feat lucky number so i megre back to main

File: composer.json [hunk 2]
```diff
@@ -30,7 +30,8 @@
         "symfony/validator": "6.1.*",
         "symfony/yaml": "*",
         "twig/extra-bundle": "^2.12|^3.0",
-        "twig/twig": "^2.12|^3.0"
+        "twig/twig": "^2.12|^3.0",
+      "ext-http": "*"
```
Comment (bangdinhnfq) [inline composer.json]: spacing

**Pull request merged**